spec/bug_report_templates_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe 'bug_report_templates' do
+  subject do
+    env_for_report = Bundler.clean_env.merge('ACTIVE_ADMIN_PATH' => active_admin_root)
+    Bundler.send(:with_env, env_for_report) do
+      Dir.chdir(chdir_path) do
+        system(Gem.ruby, template_path)
```

Comment:
I think you can pass the desired env to `system` as the first parameter. I think that might result a bit simpler.